1

```diff
@@ -0,0 +1,18 @@
+Remove functions deprecated a year ago
+
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+#
+# Date:      Wed Jul 29 12:46:41 2020 +1200
+#
+# On branch cache_deprecate
+# Your branch is ahead of 'origin/master' by 1 commit.
+#   (use "git push" to publish your local commits)
+#
+# Changes to be committed:
+#	modified:   CRM/Core/BAO/Cache.php
+#	modified:   CRM/Utils/System.php
+#
+# Untracked files:
+#	Civi/Api4/Action/Contact/Merge.php
+#
```

Comment:
Oops.

But FYI I haven't tried running yet

Reply:
It has magically disappeared

Reply:
What has? I don't see anything. (grin)